screens/Messages/MessagesList.jsx

```diff
@@ -37,17 +38,24 @@ const MessagesList = () => {
   ]);
 
   useEffect(() => {
-    setValue('');
-  }, [messages.length]);
+    if (value.length > 0 && value.trim() !== '') {
+      setSendMessages(true);
+    } else {
+        setSendMessages(false)
+    }
```

Comment:
you can write this in one line, like this:

`setSendMessages(value.length > 0 && value.trim() !== '');`

Reply:
Have changed.